app/services/qna.py

```
@@ -122,10 +168,104 @@ async def send_qna_summary_email(self, to_email: EmailStr, qna_pkg: ProjectQnAPa
+			order = await self._cohere_rerank_optional(merged_query, texts, top_n=min(len(texts), top_k))
+			texts = [texts[i] for i in order]
+			file_names = [file_names[i] for i in order]
+		except Exception:
+			logging.exception("Rerank failed; using vector order")
+		
+		# 6) prepend README as primer (if exists)
+		readme = await self._readme_blob(user_id, str(repo_id), salt_b64)
+		parts = []
+		if readme:
+			parts.append("### FILE: README ###\n" + readme)
+		for fn, t in zip(file_names, texts):
```

Ruff (0.14.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)